### ISSUE TITLE
Blockbase: Unwanted top margin for paragraph

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/themes#5617]
## Steps to replicate

* Add a Group block
* Add a Paragraph block inside the Group block
* Add paddings to the Group block
* See it on the front-end

For example:
```
<!-- wp:group {"style":{"spacing":{"padding":{"top":"8px","right":"8px","bottom":"8px","left":"8px"}}},"backgroundColor":"primary","textColor":"background"} -->
<div class="wp-block-group has-background-color has-primary-background-color has-text-color has-background" style="padding-top:8px;padding-right:8px;padding-bottom:8px;padding-left:8px"><!-- wp:paragraph -->
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin id arcu aliquet, elementum nisi quis, condimentum nibh. Donec hendrerit dui ut nisi tempor scelerisque.</p>
<!-- /wp:paragraph --></div>
<!-- /wp:group -->
```
## Result

There is an unwanted top margin for the Paragraph block on the front-end. There is no margin in the editor as expected.
This is similar to #5616. The CSS for a Quote block is messing this up.

**Front-end**
<img width="1715" alt="Screenshot 2022-03-03 at 19 30 10" src="https://user-images.githubusercontent.com/908665/156639953-3eb0ddd2-054a-4a67-a28d-92ffce824b07.png">

**Editor**
<img width="1715" alt="Screenshot 2022-03-03 at 19 30 21" src="https://user-images.githubusercontent.com/908665/156639878-bb545300-2372-4ff3-8494-9e19e80ac284.png">

## Expected

No margin for the paragraph block.

**Comment by iamtakashi** [comment]
I saw a PR #5637 that fixes one of the cases with a Group block, but the issue seems to happen with many other blocks such as Columns, Media & Text, Cover, Layout Grid, etc. Can we also fix it with other blocks?

**Comment by iamtakashi** [comment]
Any update on this issue, by any chance? Some patterns like this look bad with Blockbase for some time. Twenty Twenty Two doesn't seem to share this issue. Can we find some solution from the theme?

<img width="1230" alt="Screenshot 2022-04-04 at 17 14 12" src="https://user-images.githubusercontent.com/908665/161588064-7d10db68-49a7-4798-b092-7f574837d97d.png">

**Comment by pbking** [comment]
This should be addressed now in #5811. @iamtakashi are you able to confirm?

**Comment by iamtakashi** [comment]
Yes, I can confirm the issue is no longer seen. Thank you all so much!!
